src/components/Hightlight/index.tsx

```diff
@@ -0,0 +1,45 @@
+import * as React from 'react';
+import styled, { themed } from '../../utils/styled';
+import { StandardProps } from '../../common';
+
+export interface HighlightProps extends StandardProps {
```

Comment:
I would also add the prop to change the color of a highlighted text and do not rely on global theme variables.

Reply:
Would it be ok to do something like: `theme={{highlightColor: '#f00'}}`?

Reply:
Fine, but as I mentioned before I don't like an idea to use **global** `theme` for very specific parts. The global theme should contain something that can be used across multiple components. For instance `theme.primary[Background]Color`, which can be used as (background) color of a button, header, footer, link, whatever else.

Reply:
![image](https://user-images.githubusercontent.com/23080979/68868254-823dd300-06f7-11ea-8a12-bd2aef5dd394.png)

That would be an example on how to do it

Reply:
PS: I don't know which thoughts other contributors have regarding this topic, therefore I can't force you changing it now.

@boromo @IKolosynskyi @guiherzog

Reply:
I took the idea from other components, like the Accordion which does this same thing, just wanted to keep standard. Additionally, the color should be set on a global level to make sure in our implementation we have a standard look across different products.

Reply:
I believe that it makes sense to have it as theme color since in case one wants to change the theme, he would probably like to change the highlight as well.